docs/spec/connection-handshake/L2-tla/Environment.tla

```diff
@@ -134,17 +159,17 @@ chmB == INSTANCE ICS3Module
     msg to either of the two chains (or both).
 
  *)
-InitEnv ==
+InitEnv ==            
     /\ \/ /\ inBufChainA \in {<<msg>> : (* ICS3MsgInit to chain A. *)
                         msg \in InitMsgs(ChainAConnectionEnds, ChainBConnectionEnds)}
           /\ inBufChainB = <<>>
        \/ /\ inBufChainB \in {<<msg>> : (* ICS3MsgInit to chain B. *)
-                        msg \in InitMsgs(ChainBConnectionEnds, ChainAConnectionEnds)}
+                        msg \in InitMsgs(ChainAConnectionEnds, ChainBConnectionEnds)} 
```

Comment:
I'm not sure this is right. Chain B should be receiving some Init message where the `localEnd` field is set to a connection end for chain B (not chain A).